testing/validate/export.js

```diff
@@ -13,7 +13,8 @@
     'dexes',
     'lending',
     'payments',
-    'assets'
+    'assets',
+    'staking',
```

Comment:
```suggestion
```

I think adding a new category to DeFi pulse adds unnecessary friction, especially since staking is not our primary service.